rts/System/Threading/SpringThreading.h

```diff
@@ -34,12 +32,7 @@ namespace spring {
 	typedef CriticalSection recursive_mutex;
 	typedef win_signal signal;
 	typedef std::condition_variable_any condition_variable;
-#elif defined(__APPLE__) || !defined(USE_FUTEX)
```

Comment:
Shouldn't this chunk have stayed and only the condition changed to `!defined(USE_FIUTEX)`?